contxt/cli/commands/clusters.py

```diff
@@ -0,0 +1,120 @@
+from typing import List, Optional
+
+import click
+import yaml
+from requests.exceptions import HTTPError
+
+from contxt.cli.clients import Clients
+from contxt.cli.utils import fields_option, print_table, sort_option
+from contxt.models.contxt import Cluster
+
+
+def _get_org(clients: Clients, name: str):
+    organization = clients.contxt.get_organization_with_name(name)
+    if not organization:
+        raise click.ClickException(f"Organization {name!r} does not exist.")
+    return organization
+
+
+@click.group()
+def clusters() -> None:
+    """Clusters"""
+
+
+@clusters.command()
+@click.argument("cluster_slug", required=False)
+@click.option("--org", required=True, help="Org slug")
+@fields_option(default=["id", "host", "slug"], obj=Cluster)
+@sort_option(default="id")
+@click.pass_obj
+def get(clients: Clients, org: str, fields: List[str], sort: str, cluster_slug: Optional[str]) -> None:
+    """Get clusters"""
+    organization = _get_org(clients, org)
+    items = (
+        [clients.contxt_deployments.get_cluster(organization.id, cluster_slug)]
+        if cluster_slug
+        else clients.contxt_deployments.get_clusters(organization.id)
+    )
+    print_table(items=items, keys=fields, sort_by=sort)
```

Comment:
is `cluster_slug` adding value here? if not:
```suggestion
    organization = _get_org(clients, org)
    items = clients.contxt_deployments.get_clusters(organization.id)
    print_table(items=items, keys=fields, sort_by=sort)
```

Reply:
yeah it's the same info but only one record instead of all. Might be useful to allow filtering by some string rather than returning a specific cluste? Not sure if this falls under YAGNI though